src/views/virtualmachines/details/tabs/snapshots/components/list/SnapshotList.tsx

```diff
@@ -39,6 +44,14 @@ const SnapshotsList: React.FC<SnapshotsListProps> = ({
         columns={columns}
         Row={SnapshotRow}
         rowData={{ restores: restoresMap, isVMRunning }}
+        NoDataEmptyMsg={() => (
+          <>
+            <Bullseye>
+              <SearchIcon />
+            </Bullseye>
+            <Bullseye>{t('No snapshots found')}</Bullseye>
+          </>
```

Comment:
consider using:
```
<EmptyState variant={EmptyStateVariant.xs}>
  <EmptyStateIcon icon={SearchIcon} />
  <EmptyStateBody>
    {t('No snapshots found')}
  </EmptyStateBody>
</EmptyState>
```

Reply:
Great! That's what I needed to figure out not to use Bullseye twice.